### ISSUE TITLE
Submit not triggering?

### ISSUE THREAD

**Issue by kkashou** [jquery-validation/jquery-validation#1960]
Whenever a user tries to click the submit button, the validate does not trigger. Is there something I can be missing ? 

```js
var container = $('.email-form .errorContainer');
$('#email').validate({
    errorContainer: container,
    errorLabelContainer: $('ol', container),
    wrapper: 'li',
    rules: {
        emailTo: {
            email: false,
            required: true,
            maxcount: true,
            multiemail: true
        }
    },
    messages: {
        emailTo: {
            maxcount: 'You can only have up to 5 email addresses.',
            multiemail: 'Invalid email format: please use a comma to separate multiple email addresses.'
        }
    },
    highlight: function (element, errorClass, validClass) {
        $(element).addClass(errorClass);
        $(element).parent().addClass(errorClass);
    },
    unhighlight: function (element, errorClass, validClass) {
        $(element).removeClass(errorClass);
        $(element).parent().removeClass(errorClass);
    },
    submitHandler: function (form) {
        $('.errorContainer').html('');
        $('.successContainer').html('');
        $.ajax({
            type: $(form).method,
            url: $(form).action,
            data: $(form).serialize(),
        }).done(function () {
            $('.successContainer').addClass('is-active').html('Success!');
            $(form)[0].reset();
        }).fail(function () {
            $('.errorContainer').addClass('is-active').html('Failed!');
        });
    }
});
```
```html
<form id="email" class="email-form" action="#" method="get">

    <h2 class="email-form__headline">Email</h2>

    <div class="errorContainer">
        <svg class="icon icon-error"><use xlink:href="#icon-error"></use></svg>
        <div class="errorContainer__content">
            There was a problem!
            <ol>
                <li>
                    <label for="emailTo" class="error">Please enter valid email address</label>
                </li>
                <li>
                    <label for="emailFrom" class="error">Please enter valid email address</label>
                </li>
            </ol>
        </div>
    </div>

    <div class="email-form__form-item">
        <div class="email-form__form-items">
            <label class="label-title" for="emailTo">To <span class="info">(Up to 5 Email Address, Comma Separated)</span></label>
            <div class="textfield-group">
                <svg class="icon icon-error-2"><use xlink:href="#icon-error-2"></use></svg>
                <input id="emailTo" class="textfield" type="email" name="emailTo" required>
            </div>
        </div>

        <div class="email-form__form-items">
            <label class="label-title" for="emailFrom">Your Email Address</label>
            <div class="textfield-group">
                <svg class="icon icon-error-2"><use xlink:href="#icon-error-2"></use></svg>
                <input id="emailFrom" class="textfield" type="email" name="emailFrom" required>
            </div>
        </div>
    </div>

    <div class="email-form__submisson">
        <input class="submit" type="submit" value="Send">
    </div>

    <div class="successContainer"></div>

</form>
```

**Comment by Arkni** [comment]
Please provide a [JSFiddle](https://jsfiddle.net/) or [JSBin](https://jsbin.com/) containing a set of reproducible steps.

**Comment by kkashou** [comment]
It was my fault ... I had a click event on the modal that intercepted the click event on the submit button. Sorry for the bother.
